storybook/src/Omar/Organisms/ImagePoll/ImagePoll.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
```

Comment:
![imagePoll-Omar_pr](https://user-images.githubusercontent.com/27885879/106358876-92030080-6317-11eb-8566-1d6270683cd3.png)
- This is how the organism looks like. Please adjust the styling.
- When I click on any tab, the 1st one only gets selected. Please implement selecting any of them. 
- The post button should be disabled by default 
- Please write a behavioral test using react-testing-library to the organism.

Reply:
I adjusted the styling.
the post button is not disabled by default.